handsontable/src/__tests__/editorManager.spec.js

```diff
@@ -93,5 +93,26 @@ describe('editorManager', () => {
         });
       });
     });
+
+    describe('should not throw an error', () => {
+      [
+        ['f2'],
+        ['backspace'],
+        ['delete'],
+        ['enter'],
+        ['enter', 'shift']
+      ].forEach((key) => {
+        it(`if ${key.join(', ')} was pressed`, () => {
+          handsontable();
+
+          selectCell(0, 0);
+          deselectCell(0, 0);
+
+          expect(() => {
+            keyDownUp(key);
+          }).not.toThrowError();
+        });
+      });
+    });
```

Comment:
```suggestion
    using('shortcut key', [
      ['f2'],
      ['backspace'],
      ['delete'],
      ['enter'],
      ['enter', 'shift']
    ], (shortcutKey) => {
      it('should not throw an error when pressed in case when the table is not selected', () => {
        handsontable();

        selectCell(0, 0);
        deselectCell(0, 0);

        expect(() => {
          keyDownUp(shortcutKey);
        }).not.toThrowError();
      });
    });
```
I'd propose rewriting that tests using the `using` syntax. 
Pros:
 * Each keyboard shortcut has its own test. The tests are independent;
  ![image](https://github.com/handsontable/handsontable/assets/571316/a61f02a2-367d-40b4-a1bb-aa2673a07f00)
 * No messing with string concatenation;

Cons:
 * None? 🤔 😄